api-2.1/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationInterpretationTranslatorImpl_2_1.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import java.util.Collections;
+
+import org.apache.commons.lang.WordUtils;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.openmrs.Obs;
+import org.openmrs.annotation.OpenmrsProfile;
+import org.openmrs.module.fhir2.FhirConstants;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+@Primary
+@Component
+@OpenmrsProfile(openmrsPlatformVersion = "2.1.* - 2.3.*")
+public class ObservationInterpretationTranslatorImpl_2_1 extends ObservationInterpretationTranslatorImpl {
+	
+	@Override
+	public CodeableConcept toFhirResource(Obs obs) {
+		if (obs.getInterpretation() == null) {
+			return null;
+		}
+		CodeableConcept interpretation = new CodeableConcept();
+		switch (obs.getInterpretation()) {
+			case NORMAL:
+				interpretation.setCoding(Collections.singletonList(addIntepretationCoding("N", obs.getInterpretation())));
+				interpretation.setText(obs.getInterpretation().toString().toLowerCase());
```

Comment:
I think we should do the same thing here that we do to translate the interpretation into it's display value.